Tweak some text alignment (#543)
hackclub/site: pages/content/sponsorship.mdx

import Letterhead from '../../components/letterhead'
import Announcement from '../../components/announcement'

<Letterhead
title="Sponsorship"
desc="Looking to get hackathon/nonprofit sponsorship from Hack Club? Here’s the rundown."
img="https://workshop-cards.hackclub.com/Sponsorship.png?brand=HQ&fontSize=300px"
path="sponsorship.mdx">

<Announcement
as="a"
copy="Is your hackathon in person and happening this semester?"
caption="$500 grants and more, with support from Hack Club and FIRST®."
href="/hackathons/grant"
iconLeft="event-code"
color="primary"
sx={{
    maxWidth: 'container',
    mx: '0',
    mt: '0',
    background: '#f9fafc !important',
    boxShadow: '0 4px 8px rgb(0 0 0 / 13%)'
}}
/>

_(To summarize: we’re a donor-supported nonprofit so outside of planned grant programs, we don't provide financial support to hackathons/other nonprofits. However, there are all kinds of ways we’d love to help out your project, so read on.)_

> [Hack Club](https://hackclub.com/) is an independent nonprofit, supported by donors, with a responsibility to our supporters to spend our budget directly on our core programs. Therefore, as much as we’d like to, **we’re not in a position to provide financial support to hackathons/other nonprofits**. Although, we might run official grant programs, like a [$500 grant for all IRL high school hackathons](https://hackclub.com/hackathons/grant). We wish you the best of luck in your sponsorship search—we recommend [Megan Cui’s “Meginar”](https://youtu.be/tOmXzA4reTY) and [Lachlan Campbell’s Flagship talk](https://notebook.lachlanjc.com/2020-01-19_how_to_start_your_first_hackathon/) if you’re looking for advice.

Reviewer: ```suggestion
[Hack Club](https://hackclub.com/) is an independent nonprofit, supported by donors, with a responsibility to our supporters to spend our budget directly on our core programs. Therefore, as much as we’d like to, **we’re not in a position to provide financial support to hackathons/other nonprofits**. We occasionally run grant programs, like the [$500 grant for IRL high school hackathons](https://hackclub.com/hackathons/grant). We wish you the best of luck in your sponsorship search—we recommend [Megan Cui’s “Meginar”](https://youtu.be/tOmXzA4reTY) and [Lachlan Campbell’s Flagship talk](https://notebook.lachlanjc.com/2020-01-19_how_to_start_your_first_hackathon/) if you’re looking for advice.
```
